Sources/Workspace/DefaultPluginScriptRunner.swift

```diff
@@ -467,13 +467,9 @@ public struct DefaultPluginScriptRunner: PluginScriptRunner, Cancellable {
         process.environment = ProcessInfo.processInfo.environment
 #if os(Windows)
         let pluginLibraryPath = self.toolchain.swiftPMLibrariesLocation.pluginLibraryPath.pathString
-        var env = ProcessInfo.processInfo.environment
-        if let Path = env["Path"] {
-            env["Path"] = "\(pluginLibraryPath);\(Path)"
-        } else {
-            env["Path"] = pluginLibraryPath
-        }
-        process.environment = env
+        var env = Environment.current
```

Comment:
Sorry, I had this on auto-merge and missed it.

I considered it but I think this is a trickier behavior than usual to test for, because it's a fundamental rule of accessing the environment that we should always be going through the `Environment` abstraction, rather than some kind of one-off bug.

The Environment behavior itself is already covered by tests and I don't think it'd be realistic to add a test for every possible place that we _access_ the environment because it would end up being like whack-a-mole every time a new issue crops up.